packages/vuetify/src/composables/__tests__/border-radius.spec.ts

```diff
@@ -0,0 +1,36 @@
+// Utilities
+import { makeBorderRadiusProps, useBorderRadius } from '..'
+import { mount } from '@vue/test-utils'
+
+// Types
+import type { BorderRadiusProps } from '..'
+
+describe('border-radius.ts', () => {
+  it('should warn if value is incorrect', () => {
+    const TestComponent = {
+      props: makeBorderRadiusProps(),
+      template: '<div/>',
+    }
+
+    mount(TestComponent, {
+      props: { rounded: 'foo' },
+    })
+
+    expect('[Vue warn]: Invalid prop: custom validator check failed for prop "rounded".').toHaveBeenTipped()
+  })
+
+  it.each([
+    [{}, []],
+    [{ rounded: 'tile' }, ['rounded-0']],
+    [{ rounded: true }, ['rounded']],
+    [{ rounded: '0' }, ['rounded-0']],
+    [{ rounded: false }, ['rounded-0']],
```

Comment:
There's [no longer `rounded-circle`, `rounded-shaped`, `rounded-pill`, except](https://github.com/vuetifyjs/vuetify/pull/12801/commits/3acaffb0#diff-cc7fa17c229cefe1de91fb3f63dd9ebea204d315a428c92218ead5f58c9a9097R17-R19).

That's cool, I felt having words soup here.